sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -3554,7 +3554,7 @@ class Dataset[T] private[sql](
   private[sql] def pythonMapInArrow(func: PythonUDF, isBarrier: Boolean = false): DataFrame = {
```

Comment:
shall we rename this `pythonMapInArrow` as well? or we can make it in separate pr

Reply:
LGTM